docs/CONFIGURATION.md

```diff
@@ -229,23 +271,25 @@ graph LR;
 ### Pod-Level Credentials
 
 > [!WARNING]
-> To enable Pod-Level credentials on K8s clusters <1.30, you need to pass `node.podInfoOnMountCompat.enable=true` into
+> To enable Pod-Level credentials on Kubernetes clusters <1.30, you need to pass `node.podInfoOnMountCompat.enable=true` into
```

Comment:
Are we thinking of removing this restriction for CSI Driver v2?

Reply:
Good catch! It's actually already removed https://github.com/awslabs/mountpoint-s3-csi-driver/commit/95272709ba0b6b6bc3451a6c985f6e85fa757514